src/app/docs/sort/sort-docs.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-docs-demo-page
+  moduleName="SkySortModule"
+  moduleSourceCodePath="src/app/public/modules/sort/"
+  pageTitle="Sort"
+>
+  <sky-docs-demo-page-summary>
+    The sort component allows users to select sorting criteria. The component creates a button and dropdown menu to provide sorting options.
```

Comment:
For consistency with the tightened up intro paragraphs, I'd change this to:
```The sort component provides a button and dropdown menu for users to select sorting criteria.```

Reply:
Done.